lib/div-alg.gi

```diff
@@ -2166,10 +2166,12 @@ local m,A,B,n1;
 
 #A:=SimpleComponentOfGroupRingByCharacter(F,G,n);
```

Comment:
Don't need this commented out line?

Reply:
It's the old code for this command.  Better leave it in case Angel doesn't like how I have revised this command.

Reply:
Ideally, @angeldelriomateos should look at https://github.com/gap-packages/wedderga/pull/66/files, inspect diffs, and review your changes. In this case there will be no need to revisit this PR later to remove the commented out line before merging.

Also, that is not clear (to me at least) without an additional comment...